zaas-client/build.gradle

```diff
@@ -1,13 +1,18 @@
 dependencies {
+    implementation project(':apiml-common')
```

Comment:
I am sorry that this is not written somewhere, but it's something relevant. A bit of history: We used to publish a library that was `security-client-spring` which offered shared spring security classes, and basically all of zaas client's functionality. It was dependend on other libraries from our project. The outcome was that it was not easy for consumers to include because 1) it dragged a lot of dependencies and 2) our use of component scan made it difficult to even build. And that's why `zaas-client` was created. To provide a simple client with minimal dependencies for extenders to consume without need to worry about depending on anything from api-layer. In the light of this fact, the inclusion of apiml-common goes against that principle. I totally get the desire to use shared code, but if we want to isolate this module, I think it's ok to accept some degree of code duplication.

Reply:
Makes sense, thanks for catching this. I removed the dependency in `zaas-client` and added the duplication of taking the token from the request.